go/go.mod

```diff
@@ -6,13 +6,14 @@ require (
 	github.com/Shopify/sarama v1.34.1
 	github.com/golang/protobuf v1.5.2
 	github.com/hashicorp/go-version v1.5.0
+	github.com/hyperledger-twgc/tape v0.2.2
```

Comment:
I believe the reference to tape here is to reuse those codes copied from fabric to tape. Per the conversation we discussed before, this is not a good idea since code block gap will grow as time goes on. Instead, I think tape 
can take some exported modules here as dependencies.

Reply:
yep, so far we reused "crypto" related for sign and "endorsement" related for peer connection.
for this, I suppose so far so good.

Reply:
I suggest we create an issue and point out this as todo item.